3rdParty/velocypack/include/velocypack/Iterator.h

```diff
@@ -190,10 +194,16 @@ struct ObjectIteratorPair {
   Slice const value;
 };
 
-class ObjectIterator : public std::iterator<std::forward_iterator_tag, ObjectIteratorPair> {
+class ObjectIterator {
  public:
   using ObjectPair = ObjectIteratorPair;
 
+  using iterator_category = std::forward_iterator_tag;
+  using difference_type = std::ptrdiff_t;
+  using value_type = ObjectIteratorPair;
+  using pointer = ObjectIteratorPair*;
+  using reference = ObjectIteratorPair&;
+
```

Comment:
does this need to be ported to velocypack main repo?

Reply:
Yes.